routers/user/home.go

```diff
@@ -447,10 +447,44 @@ func Issues(ctx *context.Context) {
 		opts.MentionedID = ctxUser.ID
 	}
 
-	counts, err := models.CountIssuesByRepo(opts)
-	if err != nil {
-		ctx.ServerError("CountIssuesByRepo", err)
-		return
+	var forceEmpty bool
+	var issueIDsFromSearch []int64
+	var keyword string
+	if !isPullList {
```

Comment:
Why not pull requests too? Are they not indexed? 🤔

Reply:
I think this is just because the scope of the change is for issues #2434

But I could see that this functionality would be useful for pull requests. I'm not sure the typical way things are done here...I guess some projects are strict about not expanding scope whereas others don't mind a bit. Any thoughts?

Reply:
Oh, and I don't know if PRs are indexed or not

Reply:
I'm just going to mark this resolved. I limited the scope to issues based on my understanding of #2434

Reply:
In the database, pull requests are issues as well (only with `is_pull = true`), they are stored in the `issue` table and they are indexed indeed (the conversations, not the code itself). This page is used for issues and PRs equally, so I think there's no reason to skip this block when `isPullList` is `true`. The distinction is already taken care of when the `IsPull` flag is set a few lines above.

Reply:
I agree. I updated and it now shows on the pulls page and seems to work just fine